.travis.yml

```diff
@@ -332,3 +332,4 @@ jobs:
 branches:
   only:
     - master
+    - wilhuff/fix-check
```

Comment:
Revert

Reply:
Sorry, was moving too fast and didn't see this. Reverted in #2979.